tools/container_run_ci

```diff
@@ -71,7 +71,10 @@ fi
 cd /opt
 ./tools/install-new-deps.sh
 export CI=1 WITH_COVER_OPTIONS=1 CHECK_GIT_STATUS=1 PERL_TEST_WARNINGS_ONLY_REPORT_WARNINGS=$PERL_TEST_WARNINGS_ONLY_REPORT_WARNINGS
-cmake -G Ninja -DCMAKE_BUILD_TYPE=Release -S . -B \"$build_dir\"
```

Comment:
If we just use `make` here we should remove the `build_dir` variable at the top. Not sure about the `target` variable. We might still need it.

Note that your newly introduce script could also have its own CMake target so we wouldn't need to change this code (and could just invoke this target like others).